modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/jdbc/JdbcQueryCursor.java

```diff
@@ -86,4 +88,10 @@ public SqlQueryType queryType() {
     public ResultSetMetadata metadata() {
         return cur.metadata();
     }
+
+    /** {@inheritDoc} */
+    @Override
+    public @Nullable HybridTimestamp observableTimestamp() {
+        return cur.observableTimestamp();
```

Comment:
I think leaking internals into public API is not a good idea.
What should use do with this?

Reply:
I see two options:
- remove inheritance from SqlAsyncCursor
- add something like BaseSqlAsyncCursor interface with public methods.

What do you think?
p.s. I updated PR with option 1 (removed inheritance).

Reply:
As was discussed privately - JdbcQueryCursor is not a part of the public API.

Reply:
@AMashenkov can the 'change request' closed?